docker-compose/test.sh

```diff
@@ -10,7 +10,7 @@ echo "running tests"
 #build latest image
 docker build -t timescale/promscale:latest ../ --file ../build/Dockerfile
 
-docker-compose -p test_docker-compose up -d
+docker compose -p test_docker-compose up -d
```

Comment:
With Compose V2, its just an alias to this. I basically did this to remove the need to install that alias.

>Compose V2 integrates compose functions into the Docker platform, continuing to support most of the previous docker-compose features and flags. You can test the Compose V2 by simply replacing the dash (-) with a space, and by running docker compose, instead of docker-compose.

https://docs.docker.com/compose/cli-command/